jans-core/document-store/src/main/java/io/jans/service/document/store/service/OxDocument.java

```diff
@@ -0,0 +1,150 @@
+package io.jans.service.document.store.service;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import io.jans.orm.annotation.AttributeName;
+import io.jans.orm.annotation.DataEntry;
+import io.jans.orm.annotation.ObjectClass;
+import io.jans.orm.model.base.Entry;
+
+/**
+ * oxDocument
+ * 
+ * @author Shekhar L. Date: 01.10.2022
+ */
+
+
+@DataEntry(sortBy = { "displayName" })
+@ObjectClass(value = "oxDocument")
+@JsonInclude(Include.NON_NULL)
+public class OxDocument extends Entry implements Serializable {
```

Comment:
In jans we removed ox prefix. It's replace with jans prefix instead.
Lets rename `OxDocument` with just `Document`. It's inside `document.store.service` package so it's clear to what functionality it belongs. Object Class can be renamed `oxDocument` -> `jansDocument`. Please add changes in `jans_schema.json`. It seems to be missed in this PR.